hugo/content/docs/cluster/db-identity-lookup.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: docs.hbs
+title: DB Identity Lookup (.NET)
+---
+
+# DB Identity Lookup
+
+This strategy uses external database to keep information about spawned actors in the cluster. Actor's placement is first checked in the cluster cache. If it is not present there then is tried to be grabbed from the database. If actor spawn information is not found there then based on actor placement strategy, new actor spawn is done. After successful placement, information about it is saved in the databse.
+
+![DB-Identity-Lookup](images/db-identity-lookup.jpg)
+
+## Async Semaphore
+
+To avoid too many parallel calls to the databse, that might potentially kill database or resulted in client's timeouts, it was needed to introduce mechanism that limits concurrency. Proto.Actor has own implementation of [AsyncSemaphore](https://github.com/asynkron/protoactor-dotnet/blob/dev/src/Proto.Actor/Utils/AsyncSemaphore.cs#L12) that protects before that. DB storage implementations have a parameter that controls concurrency level.
```

Comment:
databse -> database

Reply:
"To avoid too many parallel calls to the database, that might potentially kill it or cause client timeouts, it can be needed to introduce a mechanism that limits concurrency. "